orcid-web/src/main/webapp/static/javascript/ng1Orcid/require.js

```diff
@@ -1,12 +1,36 @@
-//npm install -g typescript
+__webpack_public_path__ = window.resourceBasePath;
+
+require("../../css/glyphicons.css");
+require("../../css/social.css");
+//  require("../../css/filetypes.css");
+// require("../../css/nova-light/theme.css");
+// require("../../css/primeicons.css");
+// require("../../css/primeng.min.css");
+require("../../twitter-bootstrap/3.3.6/css/bootstrap.min.css");
+require("../../css/orcid.new.css");
+require("../../css/academicons.css");
+// require("../../css/jquery-ui-1.10.0.custom.min.css");
+// require("../../css/noto-font.css");
+require("../../css/noto-sans-googlefonts.css");
+if (window.location.href.indexOf("/print") > 0) {
+  require("../../css/orcid-print.css");
+}
 
 function requireAll(requireContext) {
   return requireContext.keys().map(requireContext);
 }
 
-require('./app/polyfills.ts');
-require('./app/main.ts');
+require("./app/polyfills.ts");
+require("jquery");
+// require("../jqueryui/1.10.0/jquery-ui.min.js");
+require("../jquery-migrate/1.3.0/jquery-migrate-1.3.0.min.js");
```

Comment:
@lizkrznarich The only use I did find for jquery-migrate-1.3.0.min.js was here 

https://github.com/ORCID/ORCID-Source/blob/master/orcid-web/src/main/webapp/static/javascript/script.js#L481-L526
for showing the "old browser alert".

Using the obsolete `$.browser` jquery attribute. We might migrate this functionality with a modern approach and remove this library.